contributing-guides/style-guide.md

```diff
@@ -4,29 +4,36 @@ This page lists specific formatting instructions for `tldr` pages.
 
 ## Layout
 
-The basic format of each page should match the following template:
+> :bulb: Placeholder syntax used in this document: `{{tip}}`.
 
-```
-# command-name
+The basic format of each page should match the following template and have at most 8 code examples:
+
+```md
+# {{command_name}}
```

Comment:
The token syntax isn't meant to be used here. Same goes for all the other places it's used outside of code blocks.